src/components/common/file-uploader/file-drop.js

```diff
@@ -24,12 +24,13 @@
  */
 import PropTypes from 'prop-types';
 import React from 'react';
+import window from 'global/window';
 
 class FileDrop extends React.PureComponent {
   static isIE = () =>
-    window &&
-    (window.navigator.userAgent.indexOf('MSIE') !== -1 ||
-      window.navigator.appVersion.indexOf('Trident/') > 0);
+    window && window.navigator &&
+      ((window.navigator.userAgent || []).indexOf('MSIE') > -1 ||
+      (window.navigator.appVersion || []).indexOf('Trident/') > -1);
```

Comment:
FY I've shifted to using [`.includes()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes)